components/script/dom/htmlmediaelement.rs

```diff
@@ -405,19 +405,19 @@ pub enum NetworkState {
 #[derive(Clone, Copy, Debug, JSTraceable, MallocSizeOf, PartialEq, PartialOrd)]
 #[repr(u8)]
 pub enum ReadyState {
-    HaveNothing = HTMLMediaElementConstants::HAVE_NOTHING as u8,
-    HaveMetadata = HTMLMediaElementConstants::HAVE_METADATA as u8,
-    HaveCurrentData = HTMLMediaElementConstants::HAVE_CURRENT_DATA as u8,
-    HaveFutureData = HTMLMediaElementConstants::HAVE_FUTURE_DATA as u8,
-    HaveEnoughData = HTMLMediaElementConstants::HAVE_ENOUGH_DATA as u8,
+    Nothing = HTMLMediaElementConstants::HAVE_NOTHING as u8,
+    Metadata = HTMLMediaElementConstants::HAVE_METADATA as u8,
+    CurrentData = HTMLMediaElementConstants::HAVE_CURRENT_DATA as u8,
+    FutureData = HTMLMediaElementConstants::HAVE_FUTURE_DATA as u8,
+    EnoughData = HTMLMediaElementConstants::HAVE_ENOUGH_DATA as u8,
```

Comment:
I'm not sure about this one, because the names are from spec.

Reply:
Ohh right, I didn't notice that. I will make the changes

Reply:
The build is now failing because you undid the rename, but didn't update the call sites. I would also add and `#![allow(...)]` directive at the head of this enum to silence the error with a comment like "# Clippy warning silenced here because these names are from the specification."

Reply:
Hi, I have made the suggested changes. The ./mach build --dev command is working in my local server but it is showing error in the PR.

Reply:
You need to run `./mach fmt` to fix formatting.